WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/ActivityIndicatorView.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+/// This is a wrapper view for using the UIActivityIndicator in SwiftUI. The native progress indicator is not available under iOS 13.
+///
+struct ActivityIndicatorView: UIViewRepresentable {
+    @Binding var isAnimating: Bool
```

Comment:
Since we are all new to `SwiftUI` I think some comments explaining how `Binding` works and how to pass a binding property would be nice. What do you think?

Reply:
I think it's a bit complex to explain how binding works and do it everywhere because it's really essential in SwiftUI. You will hardly find a view without `@Binding` properties declared. It could get a bit repetitive